src/containers/Notes/Notes.tsx

```diff
@@ -40,7 +41,13 @@ class Notes extends React.Component<{}, NotesState> {
             .then((snapshot) => {
                 if (snapshot.val()) {
                     const notes: Note[] = Object.values(snapshot.val());
-                    this.setState({ ...this.state, notes, activeNote: notes[FIRST_ELEMENT], loading: false });
+                    const sortedArrayByDate = notes.sort((a, b) => +new Date(a.date) - +new Date(b.date)).reverse();
+                    this.setState({
+                        ...this.state,
+                        notes: sortedArrayByDate,
+                        activeNote: notes[FIRST_ELEMENT],
```

Comment:
if notes is sortedArrayByDate, then active note should be sortedArrayByDate[FIRST_ELEMNT], 
not notes[FIRST_ELEMENT]